src/main/java/dev/shadowsoffire/hostilenetworks/block/LootFabBlock.java

```diff
@@ -22,8 +22,12 @@
 
 public class LootFabBlock extends HorizontalDirectionalBlock implements TickingEntityBlock {
 
-    public LootFabBlock(Properties props) {
-        super(props);
+    public LootFabBlock() {
+        super(Properties.of()
+                .lightLevel(s -> 1)
+                .strength(4, 3000)
+                .isRedstoneConductor((s, g, p) -> false)
+                .noOcclusion());
```

Comment:
Block Properties are intentionally not inlined in the Block ctor, since it makes the block non-extensible.